sql/liveqa.sql

```diff
@@ -1,9 +1,12 @@
-CREATE USER `liveqa`@`localhost` IDENTIFIED BY 'liveqa';
+CREATE USER `liveqa`@`mysql` IDENTIFIED BY 'liveqa';
```

Comment:
I will change this to `'liveqa'@'%'` to also work in podman pods, where multiple containers share localhost.